ogcapi-features/README.md

```diff
@@ -99,39 +100,43 @@ by *STAC API - Core*, the
 
 These conformance classes also requires implementation of the link relations in the [STAC API - Core](../core) conformance class.
 
+While the STAC definition does not require the existence of the `type` field in
+all Link objects, this is required by OAFeat and is thereby required by the
+*STAC API - Features* conformance class.
```

Comment:
```suggestion
All Link objects SHALL have a `type` field, as this is required by OAFeat and is thereby required by the
*STAC API - Features* conformance class.

If the target of a Link's `type` is unknown, `type` SHOULD be set to `application/octet-stream` or `text/plain`.
```

If we're keeping the direct dependency on `OGC-API - Features` (https://github.com/radiantearth/stac-api-spec/pull/368#issuecomment-1429934890), this language should be made stronger IMO.

Reply:
We use `must` instead of `shall`, but I agree with the strenth of the wording and will make a change like that.